flipperdevices/flipperzero-firmware
# [FL-2202] Reinit gauge after profile update

# What's new

- Reinit gauge after profile update

# Verification 

- Check `power_info`, ensure that capacity looks odd
- Compile and upload
- Check `power_info` once again, capacity should be ok
- Remove battery from flipper.
- Connect battery, verify correct battery profile uploaded to gauge (use `power_info`)

# Checklist (do not modify)
